include/mbedtls/aes.h

```diff
@@ -88,6 +88,18 @@ typedef struct
 }
 mbedtls_aes_context;
 
+#if defined(MBEDTLS_CIPHER_MODE_XTS)
+/**
```

Comment:
Please add references for XEX and XTS at the top of the file. (Preexisting: it's also missing references for the other modes of operation.)

Reply:
Fixed